Examples/Scripts/PreFreeSurferPipelineBatch.sh

```diff
@@ -66,8 +66,8 @@
 #
 #   Retrieve the following command line parameter values if specified
 #
-#   --StudyFolder= - primary study folder containing subject ID subdirectories
-#   --Subjlist=    - quoted, space separated list of subject IDs on which
+#   --StudyFolder= - primary study folder containing session ID subdirectories
+#   --Sessionlist=    - quoted, space separated list of session IDs on which
```

Comment:
The parsing block says `--Session`, not `--Sessionlist`, but `main()` uses a list...

Reply:
fixed in previous commit